### PR TITLE
Create 2015-08-31-intro-to-daisywheeljs.markdown

### PR DESCRIPTION
**Pull request by likethemammal**

### CODE REVIEW

File: _posts/2015-08-31-intro-to-daisywheeljs.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+After installation, whenever the Daisywheel gets loaded, the user can swiftly input text without using a single key on their keyboard.
+
+If you have trouble setting up your gamepad in the browser, remember, the Gamepad API is experimental. Take a look over the [troubleshooting section](http://daisywheeljs.org/#troubleshooting) of the main site, and try out one of the testers. Not all controllers, browsers, and operating systems are supported. To leave feedback you can hit me up on Twitter [@likethemammal](https://twitter.com/LikeTheMammal) or make an issue on [Github](https://github.com/likethemammal/daisywheeljs/issues). There are a lot of controller configurations out there and only so many can be tested at a time.
+
+### Support the project on Patreon
+
+If you like this project and want to see it continue, consider supporting it on [Patreon](patreon.com/daisywheeljs). There, you can vote on which controller will get support next and what the feature schedule will be. You'll have the opportunity to become a beta tester or join in small group chats with other knowledgeable developers.
```
Comment (akenn) [inline _posts/2015-08-31-intro-to-daisywheeljs.markdown]: https://patreon.com/daisywheeljs

File: _posts/2015-08-31-intro-to-daisywheeljs.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Adding the DaisywheelJS onscreen-keyboard to HTML5 Gamepad projects
+tags: daisywheeljs daisywheel gamepad html5
+username: likethemammal
+---
+
+### Some background
+
+HTML5 gaming has developed quite a bit since it first landed on the scene a few years back. Now, with proclamations from major browser vendors starting to pull support from Flash, and even [block some Flash ads(http://www.themonitordaily.com/google-chrome-adobe-autoplay/25687/), usage of HTML5’s latest technologies will spread further. Old Flash game devs will become new HTML5 game devs and entire games will be ported over for maintainability’s sake.
```
Comment (akenn) [inline _posts/2015-08-31-intro-to-daisywheeljs.markdown]: Forgot a closing **]**. Should be `[block some Flash ads](http://www.themonitordaily.com/google-chrome-adobe-autoplay/25687/)`

File: _posts/2015-08-31-intro-to-daisywheeljs.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+### A solution appears!
+
+If you’re a user of Steam, the massive game marketplace that has taken over PC gaming, you might have used their [Big Picture Mode](http://store.steampowered.com/bigpicture/). It’s an amazing tool and gives me more hope for the future. Basically, it turns the Steam interface into something that resembles a console and is much more controller friendly. This compared to the normal layout navigated with a mouse and keyboard. The “friends” tab of Big Picture Mode presents you with an on-screen keyboard whenever you chat. This has been deemed the “Daisywheel” (after the [old impact printing method](https://en.wikipedia.org/wiki/Daisy_wheel_printing)) And this is where our journey begins.
```
Comment (akenn) [inline _posts/2015-08-31-intro-to-daisywheeljs.markdown]: > This compared to the normal layout navigated with a mouse and keyboard.

Kill this sentence

Comment (akenn) [inline _posts/2015-08-31-intro-to-daisywheeljs.markdown]: > This has been deemed the “Daisywheel” (after the [old impact printing method (https://en.wikipedia.org/wiki/Daisy_wheel_printing)) And this is where our journey begins.

This has been deemed the “Daisywheel” (after the [old impact printing method (https://en.wikipedia.org/wiki/Daisy_wheel_printing)). This is where our journey begins.

File: _posts/2015-08-31-intro-to-daisywheeljs.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Imagine a radial keyboard, with groups of symbols around the edge, requiring only 2 “actions” (from the user) to select any character or number. This, opposed to the possible 12 “actions” that a traditional QWERTY on-screen keyboard might require (furiously arrowing from letter to letter). Just to select a single character.
+
+If your imagination died with your childhood, no worries (well, some worries), there’s a screenshot below.
```
Comment (akenn) [inline _posts/2015-08-31-intro-to-daisywheeljs.markdown]: lol

File: _posts/2015-08-31-intro-to-daisywheeljs.markdown [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+### A solution appears!
+
+If you’re a user of Steam, the massive game marketplace that has taken over PC gaming, you might have used their [Big Picture Mode](http://store.steampowered.com/bigpicture/). It’s an amazing tool and gives me more hope for the future. Basically, it turns the Steam interface into something that resembles a console and is much more controller friendly. This compared to the normal layout navigated with a mouse and keyboard. The “friends” tab of Big Picture Mode presents you with an on-screen keyboard whenever you chat. This has been deemed the “Daisywheel” (after the [old impact printing method](https://en.wikipedia.org/wiki/Daisy_wheel_printing)) And this is where our journey begins.
+
+Imagine a radial keyboard, with groups of symbols around the edge, requiring only 2 “actions” (from the user) to select any character or number. This, opposed to the possible 12 “actions” that a traditional QWERTY on-screen keyboard might require (furiously arrowing from letter to letter). Just to select a single character.
```
Comment (akenn) [inline _posts/2015-08-31-intro-to-daisywheeljs.markdown]: Traditional keyboards implemented for typing on controllers present the user with a big, full-sized QUERTY keyboard. In order to type out a message, the user has to navigate around the keyboard with the thumb-sticks on their controller, vigorously inputting anywhere from 2 to 12 actions just to enter a single character. Now imagine a world where a radial keyboard existed, with groups of symbols around the edge, requiring only 2 actions from the user (a flick of the thumb-stick and a tap on an action button) to select any character or number.

**Comment by mediocregopher** [comment]
lgtm, I agree with andrew's comments, although it looks like you fixed them already

**Comment by theabraham** [comment]
Yeah looks good!

**Comment by MarcoPolo** [comment]
:ship: :it:

**Comment by MarcoPolo** [comment]
I'm interested in a post about how it was implemented!

**Comment by likethemammal** [comment]
That'll be the next thing I write @marcopolo

**Comment by likethemammal** [comment]
@MarcoPolo heres that next article: https://github.com/cryptic-io/cryptic-chat/pull/41

**Pull request merged**